api/web_conn.go

```diff
@@ -32,7 +33,9 @@ type WebConn struct {
 }
 
 func NewWebConn(c *Context, ws *websocket.Conn) *WebConn {
-	go SetStatusOnline(c.Session.UserId, c.Session.Id, false)
+	if len(c.Session.UserId) > 0 {
```

Comment:
Just making sure. This will be true if the web client auths using a cookie, correct?